lingpy/algorithm/extra.py

```diff
@@ -191,23 +191,18 @@ def infomap_clustering(threshold, matrix, taxa=False, revert=False):
     Notes
     -----
     Infomap clustering is a community detection method originally proposed by
-    :evobib:`Rosvall2008`.
-
-    Requires the igraph package is required, downloadable from http://igraph.org/.
+    :evobib:`Rosvall2008`. This method requires the igraph package,
+    downloadable from http://igraph.org/.
     """
     if not igraph:
         raise ValueError("The package igraph is needed to run this analysis.")
     if not taxa:
         taxa = list(range(1, len(matrix) + 1))
-
     G = igraph.Graph()
-    vertex_weights = []
+    # add vertices to the graph
```

Comment:
last commit destroyed the adding of vertices to the graph, but they need to be explicitly added in igraph.